arch/risc-v/src/esp32c3/esp32c3_sha.c

```diff
@@ -668,6 +668,7 @@ void esp32c3_sha1_free(struct esp32c3_sha1_context_s *ctx)
 void esp32c3_sha256_init(struct esp32c3_sha256_context_s *ctx)
 {
   memset(ctx, 0, sizeof(struct esp32c3_sha256_context_s));
+  ctx->mode = ESP32C3_SHA2_256;
```

Comment:
Could you please do this similar change to the `esp32c3_sha1_init` function?
```c
  ctx->mode = ESP32C3_SHA1_1;
```
And remove the `esp32c3_sha256_starts` function which is not currently used in this driver nor present in the driver interface.